Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControlElement.swift

```diff
@@ -19,12 +19,15 @@ public enum MediaControlPosition {
 
 extension MediaControl {
     open class Element: UICorePlugin, MediaControlElementType {
-        open var panel: MediaControlPanel {
-            return .center
-        }
+        open var panel: MediaControlPanel { .center }
+        open var position: MediaControlPosition { .left }
+    }
+}
 
-        open var position: MediaControlPosition {
-            return .left
-        }
+extension MediaControl {
+    enum AnimationState {
+        case showing
+        case hiding
+        case none
```

Comment:
Can we put the `enum` at the beginning of the escopo of the extension, just to maintain a pattern?